packages/create-catalyst/src/utils/clone-catalyst.ts

```diff
@@ -71,29 +56,19 @@ export const cloneCatalyst = async ({
       scripts: z.object({}).passthrough().optional(),
       dependencies: z.object({}).passthrough(),
       devDependencies: z.object({}).passthrough(),
-      private: z.boolean().optional(),
-      exports: z.object({}).passthrough().optional(),
-      sideEffects: z.boolean().optional(),
-      peerDependencies: z.object({}).passthrough().optional(),
     })
     .passthrough()
     .parse(
-      merge(
-        {},
-        readJsonSync(join(projectDir, 'tmp/package.json')),
-        readJsonSync(join(projectDir, 'package.json')),
-        { name: projectName, description: '' },
-      ),
+      merge({}, readJsonSync(join(projectDir, 'package.json')), {
+        name: projectName,
+        description: '',
+      }),
     );
 
   delete packageJson.private;
   delete packageJson.exports;
   delete packageJson.sideEffects;
```

Comment:
I would audit these, in case we can remove more.

Reply:
Do these even get added? Since I removed above, I think I can just remove these, no?

Reply:
What I don't get is why we added them and then deleted.